spanner/ddl/ast_test.go

```diff
@@ -187,6 +200,17 @@ func TestPrintCreateTable(t *testing.T) {
 			"interleaved",
 			false,
 			t2,
+			"CREATE TABLE mytable (\n" +
+				"	col1 INT64 NOT NULL ,\n" +
+				"	col2 STRING(MAX),\n" +
+				"	col3 BYTES(42),\n" +
+				") PRIMARY KEY (col1 DESC),\n" +
+				"INTERLEAVE IN PARENT  ON DELETE CASCADE",
```

Comment:
Hmm the empty parent table name looks odd, this double spacing looks like an error at first.

Can we genearte a dummy spSchema so that the parent table name is also populated correctly?